Shiny LaTeX Documentation

Myoldmopar opened the pull request:
## Pull request overview

This replaces all the Markdown docs with LaTeX, and includes build and packaging rules for this source.  Other small cleanups in the CMake files along the way.  Enormous effort from @michael-okeefe and @nealkruis.
### Work Checklist

Add to this list or remove from it as applicable.  This is a simple templated set of guidelines.
- [x] Title of PR should be user-synopsis style (clearly understandable in a standalone changelog context)
- [x] At least one of the following appropriate labels must be added to this PR to be consumed into the changelog:
  - Defect: This pull request repairs a github defect issue.  The github issue should be referenced in the PR description
  - Refactoring: This pull request includes code changes that don't change the functionality of the program, just perform refactoring
  - NewFeature: This pull request includes code to add a new feature to EnergyPlus
  - Performance: This pull request includes code changes that are directed at improving the runtime performance of EnergyPlus
  - DoNoPublish: This pull request includes changes that shouldn't be included in the changelog
### Review Checklist

This will not be exhaustively relevant to every PR.
- [x] CI status: all green or justified
- [x] Documentation changes in place

michael-okeefe committed:
Remove markdown documentation source tree

michael-okeefe committed:
Add initial LaTeX documentation sources

michael-okeefe committed:
Merge in latest changes

michael-okeefe committed:
Turn off alternative fonts for now

michael-okeefe committed:
Move overview items into overview.tex

michael-okeefe committed:
Move hvac-template-object sub-headings into appropriate file

michael-okeefe committed:
Move sub-files of energyplus-economics to energyplus-economics.tex

matthew-larson committed:
Fix equations in integrated-solution-manager/basis-for-the-zone-and-air-system-integration.

michael-okeefe committed:
Consolidate group-parametrics and parametric-objects

michael-okeefe committed:
Consolidate input-for-output and clean up listings

michael-okeefe committed:
Add missing commands defined from Pandoc's LaTeX header

michael-okeefe committed:
Move sub-section into enclosing file

michael-okeefe committed:
Move input statements into their corresponding subsections

michael-okeefe committed:
Fix bad quotes around "Terrain"

matthew-larson committed:
Add equation tags to equations in integrated-solution-manager/basis-for-the-zone-and-air-system-integration.

matthew-larson committed:
Fix equations in integrated-solution-manager/generic-contaminant-predictor-corrector and add equation tags.

matthew-larson committed:
Fix equations in integrated-solution-manager/generic-contaminant-predictor-corrector and add equation tags.

matthew-larson committed:
Fix equations in integrated-solution-manager/moisture-predictor-corrector and add equation tags.

matthew-larson committed:
Fix spacing in integrated-solution-manager/zone-air-mass-flow-conservation.

michael-okeefe committed:
Fix grammatical error

michael-okeefe committed:
Fix litering of non-breaking spaces and listings that were not recognized

michael-okeefe committed:
Set listings to a smaller font size

michael-okeefe committed:
Remove mistaken listing

michael-okeefe committed:
Clean up IDF formatting

michael-okeefe committed:
Create listing for code block that was missed

michael-okeefe committed:
Tidy up IDF code

michael-okeefe committed:
Remove non-breaking space

matthew-larson committed:
Fix equations. tables, and spacing in surface-heat-balance-manager-processes.

matthew-larson committed:
Fix equations. tables, and spacing in surface-heat-balance-manager-processes.

michael-okeefe committed:
Add the framed package for callouts/warnings/admonitions

michael-okeefe committed:
Set the color of note/callout/admonition boxes

michael-okeefe committed:
Define commands/environments for callouts and warnings

michael-okeefe committed:
Fix listings, table issues, clean up code blocks

michael-okeefe committed:
Add preamble from Pandoc's default template

michael-okeefe committed:
Add comment for how to turning on "zebra stripes" for tables

Unfortunately, this does not work across the board. Or rather, it works TOO well. Declaring row striping in the preamble ends up also striping equations wrapped in a \begin{array} ... \end{array} and also does not work consistently on every table. For example, sometimes, the table caption area gets striped which looks horrible.

Zebra striping CAN be turned on per table however:

(example from https://www.sharelatex.com/learn/Tables)

{\rowcolors{3}{green!80!yellow!50}{green!70!yellow!40}
\begin{tabular}{ |p{3cm}|p{3cm}|p{3cm}|  }
\hline
\multicolumn{3}{|c|}{Country List} \\
\hline
Country Name     or Area Name& ISO ALPHA 2 Code &ISO ALPHA 3 \\
\hline
Afghanistan & AF &AFG \\
Aland Islands & AX   & ALA \\
Albania &AL & ALB \\
Algeria    &DZ & DZA \\
American Samoa & AS & ASM \\
Andorra & AD & AND   \\
Angola & AO & AGO \\
\hline
\end{tabular}
}

michael-okeefe committed:
Fix accidentally escaped text inside listing (verbatim) block

michael-okeefe committed:
Fix IDF code blocks, some equations, 1st / 2nd superscripts

michael-okeefe committed:
Add newlines to end of files

michael-okeefe committed:
Fix spacing for date

michael-okeefe committed:
Crop PNG image to remove excess whitespace

michael-okeefe committed:
Fix IDF code block indentation; remove non-breaking spaces; add line to end of file

michael-okeefe committed:
Mute the warning color a bit

michael-okeefe committed:
Add important note about fixlatex package being no longer necessary

See: https://latex-project.org/ltnews/ltnews22.pdf this package
contained critical fixes to the latex2e kernel that were not merged for
sake of backward compatibility. However, after over 20 years, they've finally
merged these changes into the latex2e kernel proper.

matthew-larson committed:
Fix table sizes, format, and symbols.

matthew-larson committed:
Wrap equations.

matthew-larson committed:
Fix tables and equations.

michael-okeefe committed:
Add collage of broken figures

This fixes the old figures 15 and 16 (images 36, 38, 39, and 40). Those images have been removed and a graphics editor was used to compose a single image.

michael-okeefe committed:
Add fixes for IDF code blocks, broken figures, broken equations, and other minor edits

This is focused on overview/group-surface-construction-elements.tex

Myoldmopar committed:
Add CMake commands for LaTeX builds, move doc source up one level

michael-okeefe committed:
Fix mis-labeled field header

michael-okeefe committed:
Add callout environment around text

michael-okeefe committed:
Merge branch 'latex-documentation' of https://github.com/NREL/EnergyPlus into latex-documentation

michael-okeefe committed:
Move \author \date to header.tex

michael-okeefe committed:
Adjust image to not have so much whitespace

Myoldmopar committed:
Try commenting the readme.html for PackageMaker builds

Maybe this will get the package build working on Mac.  Or maybe it will break it further.  We'll see.

Myoldmopar committed:
Try using a different readme file path

In the source tree instead...a better fix hopefully

Myoldmopar commented:
## Some issues showing up as build errors

FYI @nealkruis @michael-okeefe 

Note that the text output in here may be garbled because I just did make -j 9 package.  I'm happy to dissect more tomorrow, but it looks like these messages are simply being matched by the CI error processor as python errors.  @lefticus I'll be cleaning up the errors as we go, but looking ahead, is there anything I can add to the decent_ci to enable LaTeX error parsing along with everything else?  

### Strange `division by 0` errors

Being reported on CI as python:0:0, but actually when you build it is shows this:

```
l.2728 ...own \includegraphics{media/image046.svg}                 
! Package graphics Error: Division by 0.
See the graphics package documentation for explanation.
```

Same for images:
- image289.svg
- image290.svg
- image291.svg
- image295.svg ... ok maybe for all the svg images?

### Illegal pream tokens errors

More LaTeX issues being processed as python.  Here's the overall error context:

```
l.837   \end{array}

[298] [1515

! Package array Error:  Illegal pream-token (J): `c' used.

See the array package documentation for explanation.
Type  H <return>  for immediate help.
 ...                                              

l.875    J
          _3 \\

! Package array Error:  Illegal pream-token (Q): `c' used.

See the array package documentation for explanation.
Type  H <return>  for immediate help.
 ...                                              

l.883     Q
           _3 \\
<use  "media/image1059.png" >] [1516 <use  "media/image1060.png" > [299]
<use  "media/image1063.png" >]
```

### Other build errors

There are others but I'm not looking at them too deeply tonight.

### Mac package error

It was looking for a readme.html PackageMaker resource file.  I think I have it fixed?  We'll see?

Myoldmopar committed:
Fix a typo in the CPack Readme thing

Will try another tag.

Myoldmopar committed:
Add a couple macros, one package, and some cleanup

matthew-larson committed:
Fix equations and tables.

matthew-larson committed:
Merge branch 'latex-documentation' of https://bitbucket.org/bigladdersoftware/energyplus/branches into latex-documentation

Myoldmopar committed:
Add proper dependency rules for engineering reference TeX

Myoldmopar committed:
Merge remote-tracking branch 'origin/latex-documentation-myoldmopar' into latex-documentation

Myoldmopar committed:
Add proper dependency rules for input output ref TeX

Myoldmopar committed:
Add proper dependency rules for acknowledgments and auxiliaryprograms

Myoldmopar committed:
Add proper dependency rules for ems app guide TeX

Myoldmopar committed:
Add proper rules for external interface guide, cleanup gitignore also

Myoldmopar committed:
Followon fix for ext interface rules

Myoldmopar committed:
Add proper dependency rules for getting started guide

Myoldmopar committed:
Add proper dependency rules for interface developer guide

matthew-larson committed:
Fix tables, equations, and symbols.

Myoldmopar committed:
Add proper dependency rules for module developer guide

Myoldmopar committed:
HAAAAAANDS

Myoldmopar committed:
My hands are typing words

Myoldmopar committed:
https://xkcd.com/1296/

Myoldmopar committed:
Add proper dependency rules for all tex docs

Myoldmopar committed:
Merge branch 'latex-documentation' of github.com:NREL/EnergyPlus into latex-documentation

Myoldmopar committed:
Clean install commands for pdf titles

Myoldmopar committed:
Cleanup bad macro declaration and install command

Myoldmopar committed:
Add gitignore for docs directory, clean _all_ warnings from acknowledgements

Myoldmopar committed:
Clean up build errors in aux programs doc

matthew-larson committed:
Fix tables, equations, and symbols.

Myoldmopar committed:
Cleanup table in EMS app guide, no errors in EMS guide now

michael-okeefe committed:
Fix listings add callouts

michael-okeefe committed:
Merge branch 'latex-documentation' of https://bitbucket.org/bigladdersoftware/energyplus into latex-documentation

michael-okeefe committed:
Merge branch 'latex-documentation' of https://github.com/NREL/EnergyPlus into latex-documentation

# Conflicts:
#	doc/engineering-reference/src/integrated-solution-manager/moisture-predictor-corrector.tex
#	doc/header.tex

Kept both the author / date in header and the added macros. Placed \author, \date last as per a typical LaTeX preamble.
Took the correction to equation in engineering reference from upstream which fixed the equation "double wrapping" \begin{equation}\[ => \begin{equation}.

matthew-larson committed:
Fix tables, equations, and symbols.

matthew-larson committed:
Fix tables, equations, and symbols.

michael-okeefe committed:
Fix enumerations, remove non-breaking space, correct listings

michael-okeefe committed:
Crop figure image to remove excessive whitespace

Myoldmopar committed:
Clean up last remaining LaTeX build errors in the smaller docs, IO and Eng ref to follow

matthew-larson committed:
Remove manual table formatting.

michael-okeefe committed:
Merge branch 'latex-documentation' of https://bitbucket.org/bigladdersoftware/energyplus into latex-documentation

michael-okeefe committed:
Merge branches 'latex-documentation' and 'latex-documentation' of https://github.com/NREL/EnergyPlus into latex-documentation

michael-okeefe committed:
Remove issue with non-unique label

Myoldmopar committed:
Slogging through cleaning up the EngRef from the back toward the front

matthew-larson committed:
Major equation and table fixes.

michael-okeefe committed:
Crop images with excessive whitespace in Input Output Reference

michael-okeefe committed:
Correct broken LaTeX in input output reference

- Remove inappropriate non-breaking spaces
- Wrap some verbatim itemized lists into formal \begin{itemized} etc.
- Wrap equations in \begin{equation} ... blocks
- Correct IDF block formatting
- Convert some lists to "tightlists" (lists without the spacing)

michael-okeefe committed:
Merge branch 'latex-documentation' of https://bitbucket.org/bigladdersoftware/energyplus into latex-documentation

michael-okeefe committed:
Merge branch 'latex-documentation' of https://github.com/NREL/EnergyPlus into latex-documentation

Myoldmopar committed:
Cleaning up more EngRef issues

Myoldmopar committed:
Merge branch 'latex-documentation' of github.com:NREL/EnergyPlus into latex-documentation

michael-okeefe committed:
Fix Package hyperref Warning: Suppressing empty link ...

This was caused by a pattern of \protect\hyperlink{section-1}{} getting inserted where there was a [] in the original text.

Fixed by a global search and replace to {[]}

Note: this commit also includes a few minor edits such as adding hyphenation to long EnergyPlus class names as well as replacing greek letters with their equivalent in mathmode.

michael-okeefe committed:
Convert SVG to PNG and link into documents

LaTeX can't embed SVG by default. Therefore, convert SVG to PNG and change document links.

Note: we may want to have a CMake task to auto-convert and crop SVGs to PNGs to preserve vector image manipulation capability.

michael-okeefe committed:
Remove all remaining non-breaking spaces in the text

Removed all character codes 160 (ASCII; u00A0 in unicode) from the text. Note: not referring to the literal "~" but to the non-breaking spaces that are within the text itself
Removed across all documents. 142,166 instances reported...

michael-okeefe committed:
Remove all tabs from file

Replace u0009 with "  " (2 spaces)

michael-okeefe committed:
Correct errors from the log

- Replace greek characters with math-mode  ν => \(\nu\)
- format IDF blocks
- hyphenate long EnergyPlus class names
- remove empty exponent \(^{}\)
- correct bracket balance issue in math mode
- wrap \begin{array} ... in \begin{equation} ...

matthew-larson committed:
Major equation and table fixes.

matthew-larson committed:
Major equation and table fixes.

Myoldmopar committed:
Continued progress cleaning up EngRef

michael-okeefe committed:
Fix remaining log issues with Input/Output Reference

Ignoring the Overfull/Underfull hbox warnings, addressed all other issues from the IO ref's build log:

- missing delimiters in equations
- equations only wrapped in \begin{array} (as opposed to \begin{equation} \begin{array}...)
- replaced literal greek text with math mode
- fixed some garbled subscripts by comparing to last MS Word build, especially in doc/input-output-reference/src/overview/group-fans

michael-okeefe committed:
Merge branch 'latex-documentation' of https://bitbucket.org/bigladdersoftware/energyplus into latex-documentation

michael-okeefe committed:
Merge branch 'latex-documentation' of https://github.com/NREL/EnergyPlus into latex-documentation

# Conflicts:
#	doc/engineering-reference/src/simulation-models-encyclopedic-reference-002/refrigeration-equipment.tex

michael-okeefe committed:
Remove spurious non-breaking spaces

Myoldmopar committed:
Merge develop, resolve conflict

michael-okeefe committed:
Merge branch 'latex-documentation' of https://github.com/NREL/EnergyPlus into latex-documentation

matthew-larson committed:
Fix table.

matthew-larson committed:
Log fixes for Engineering Reference.

michael-okeefe committed:
Make visual fixes through IO Ref

michael-okeefe committed:
Merge branch 'latex-documentation' of https://bitbucket.org/bigladdersoftware/energyplus into latex-documentation

michael-okeefe committed:
Add minor fixes

Myoldmopar committed:
Continued efforts on EngRef

matthew-larson committed:
Log fixes for Engineering Reference.

Myoldmopar committed:
Merge branch 'latex-documentation' of github.com:NREL/EnergyPlus into latex-documentation

michael-okeefe committed:
Merge branch 'latex-documentation' of https://github.com/NREL/EnergyPlus into latex-documentation

michael-okeefe committed:
Merge branch 'latex-documentation' of https://bitbucket.org/bigladdersoftware/energyplus into latex-documentation

Myoldmopar committed:
Plethora of fixes to EngRef, end approaching

Myoldmopar committed:
Merge branch 'latex-documentation' of github.com:NREL/EnergyPlus into latex-documentation

matthew-larson committed:
Log fixes for Engineering Reference.

michael-okeefe committed:
Add missing sections to Auxiliary Programs

Missing sections due to edge cases in processing scripts.

michael-okeefe committed:
Merge branch 'latex-documentation' of https://github.com/NREL/EnergyPlus into latex-documentation

michael-okeefe committed:
Merge branch 'latex-documentation' of https://bitbucket.org/bigladdersoftware/energyplus into latex-documentation

matthew-larson committed:
Log fixes for Engineering Reference.

matthew-larson committed:
Merge branch 'latex-documentation' of https://bitbucket.org/bigladdersoftware/energyplus/branches into latex-documentation

Myoldmopar committed:
Clean build of Engineering Referenced

Myoldmopar commented:
Remaining things that can get pushed off until later:
- [ ] Encyclopaedic Reference folders should be merged together
- [ ] Can we pull `\label`s outside of `\caption`s so that we don't need to `protect` them?
- [ ] Lots of multiply defined labels need to be cleaned up

Myoldmopar committed:
Change case of single character in compliance doc name

Myoldmopar commented:
Had to make one tiny change from UsingEnergyPlusforCompliance to UsingEnergyPlusForCompliance.  Package failed on Linux because it is case sensitive.  Should be good on Windows and Mac.

michael-okeefe committed:
Fix missing content and subscripts

michael-okeefe committed:
Merge branch 'latex-documentation' of https://bitbucket.org/bigladdersoftware/energyplus into latex-documentation

michael-okeefe committed:
Merge branch 'latex-documentation' of https://github.com/NREL/EnergyPlus into latex-documentation

# Conflicts:
#	doc/engineering-reference/src/daylighting-and-window-calculations/daylight-factor-calculation.tex
#	doc/engineering-reference/src/daylighting-and-window-calculations/time-step-daylighting-calculation.tex
#	doc/engineering-reference/src/daylighting-and-window-calculations/window-calculation-module.tex

Most merge conflicts were due to the same equations being edited but differences in indentation, etc. I tried to make a consistent editing for all indentation sections.

Myoldmopar committed:
Fix multiply defined table

Myoldmopar commented on doc/auxiliary-programs/src/weather-converter-program/weather-data-availability.tex:
```diff
@@ -6,10 +6,18 @@ \section{Weather Data Availability}\label{weather-data-availability}
 % table 1
 \begin{longtable}[c]{p{1.2in}p{1.2in}p{1.2in}p{1.2in}p{1.2in}}
-\caption{Comparison of E/E with ESP-r/DOE-2/BLAST Weather Data Formats \protect \label{table:comparison-of-ee-with-esp-rdoe-2blast-weather}}\\
+
```
@michael-okeefe @nealkruis @matthew-larson 

This is the type of change I had to make in order to get rid of the multiply defined label warnings for the tables that end up spanning two pages.  

For longtable, if we put the caption in the regular header, it will be repeated for each page the table has to span.  Because the \label is also in the caption, it will be re-declared each time as well.  If the table only hits one page, no problem, but if it spans a second page, it will be declared twice.  Or thrice.  

The solution is to put a "first header" where the caption, label, and regular header row is declared.  Then declare a regular (subsequent) header _without the label_ that is used on all subsequent pages.

It's a bit annoying, and there may be a better shorthand way to do it.  But the longtable docs are not good compared to most LaTeX package docs.  You have any thoughts moving forward?  I can try to come up with a scripted clean-up pass if needed.  But this can also wait until after release.

michael-okeefe committed:
Merge branch 'latex-documentation' of https://github.com/NREL/EnergyPlus into latex-documentation

Myoldmopar committed:
Add wkhtmltopdf and pandoc finders back in for building ValidationDocs package

Myoldmopar commented:
OK, so as I mentioned above, I think we should merge the encyclopaedic reference folders together unless there is a reason not to.  And I am also curious your thoughts on the font.  I don't mind putting it back in; I don't think it was actually causing any problems, XeLaTeX just had to do some preprocessing on each build.

michael-okeefe commented:
Yes, I definitely agree we should merge the encyclopaedic reference folders together but, if possible, can we hold off until I've pushed up the the fixes for subscripts/superscripts/tables? I think there is a general "file reorganization" activity for both the Engineering Reference and IO Reference.

In terms of fonts, I'm actually fairly neutral on the matter: anything readable. I personally like serif although Neal likes sans-serif. Feel free to try a few and go with what you like. Here are some links to some open fonts for a start:

# **Monospace**

Inconsolata: http://www.levien.com/type/myfonts/inconsolata.html

license: Open Font License

# **Serif and Sans-Serif**

Linux Libertine and Linux Biolinum: https://sourceforge.net/projects/linuxlibertine/files/linuxlibertine/5.3.0/

license: GPL

matthew-larson committed:
Fix figure format, captions, and references for Plant Application Guide.

matthew-larson committed:
Major equation and table fixes.

Myoldmopar commented:
So for reference, I'm getting positive feedback from just using Computer Modern...I'll admit it's a biased sample because most of those surveyed likely used LaTeX in grad school, so it's a familiar sight to them.    @jasondegraw also proposed using `newtxtext` and `newtxmath`, which would be a fine option.  We'll have to open up a vote.  I'll set something up.  

As for the file organization, **definitely** no rush to start moving files around.  (Although I think Git is pretty good about tracking diffs including across rename boundaries.....)

Myoldmopar commented:
OK, so here is a Google Form that will help us select a font.  Yeah, yeah, it allows multiple responses.  If you really feel that strongly about one font, abuse it.

http://goo.gl/forms/OXlGtkW71t

jasondegraw commented:
I won't stuff the ballot box, but :+1: :100: :shipit: :facepunch: for Times via `newtx`

michael-okeefe committed:
Partial way through re-incorporating missing tables, superscripts, subscripts

michael-okeefe committed:
Merge branch 'latex-documentation' of https://github.com/NREL/EnergyPlus into latex-documentation

michael-okeefe committed:
Merge branch 'latex-documentation' of https://bitbucket.org/bigladdersoftware/energyplus into latex-documentation

michael-okeefe committed:
Fix broken equations and tables causing compile log issues

michael-okeefe committed:
Continue to diff in changes for super/subscripts, new table formatting, etc.

michael-okeefe committed:
Replace broken table

michael-okeefe committed:
Merging in more changes for tables, subscripts, superscripts

matthew-larson committed:
Major equation and table fixes.

matthew-larson committed:
Fix log error.

matthew-larson committed:
Major equation and table fixes.

michael-okeefe committed:
Fix formatting bug: extra space before start of includes

michael-okeefe committed:
Fix subscripts and equations in air-system-compound-component-groups.tex

matthew-larson committed:
Fix table format.

matthew-larson committed:
Merge branch 'latex-documentation' of https://bitbucket.org/bigladdersoftware/energyplus/branches into latex-documentation

matthew-larson committed:
Fix subscripts and table headers.

matthew-larson committed:
Fix subscripts and table headers.

michael-okeefe committed:
Fix tables, include missing content, include missing super/sub-scripts

michael-okeefe committed:
Merge branch 'latex-documentation' of https://bitbucket.org/bigladdersoftware/energyplus into latex-documentation

matthew-larson committed:
Fix super/sub-scripts and table headers.

matthew-larson committed:
Fix super/sub-scripts and table headers.

michael-okeefe committed:
Fix bugs with table rendering

michael-okeefe committed:
Continue fixing tables, sub/super scripts, etc. in Engineering Reference

michael-okeefe committed:
Merge branch 'latex-documentation' of https://bitbucket.org/bigladdersoftware/energyplus into latex-documentation

michael-okeefe committed:
Restore sub/superscripts and tables.

michael-okeefe committed:
Restore missing content from indoor swimming pool section

michael-okeefe committed:
Restore super/subscripts, tables, fix equations

michael-okeefe committed:
Restore missing table that did not convert initially; fix subscripts

michael-okeefe committed:
Restore missing content

michael-okeefe committed:
Restore super/sub-scripts; fix table headers

michael-okeefe committed:
Replace super/subscripts; fix table formatting

michael-okeefe committed:
Remove newlines after toprule

michael-okeefe committed:
Wrap comment so it doesn't autowrap

The line is too long with the existing font size and font.
Manually wrap it so it looks better

michael-okeefe committed:
Restore missing content, sub/superscripts

matthew-larson committed:
Fix super/sub-scripts and table headers.

matthew-larson committed:
Merge branch 'latex-documentation' of https://bitbucket.org/bigladdersoftware/energyplus/branches into latex-documentation

matthew-larson committed:
Fix super/sub-scripts and table headers.

matthew-larson committed:
Fix super/sub-scripts and table headers for interface-developer.

Myoldmopar committed:
Merge remote-tracking branch 'origin/develop' into latex-documentation

matthew-larson committed:
Fix super/sub-scripts and table headers for module-developer.

michael-okeefe committed:
Fix equations and subscripts; restore missing content

Tried to replace this sort of phrase {\mathop m\limits^ \bullet _{subscript}} with \dot{m}_{subscript}

michael-okeefe committed:
Merge branch 'latex-documentation' of https://bitbucket.org/bigladdersoftware/energyplus into latex-documentation

michael-okeefe committed:
Merge branch 'latex-documentation' of https://github.com/NREL/EnergyPlus into latex-documentation

matthew-larson committed:
Fix table headers and add missing content for output-details-and-examples.

matthew-larson committed:
Merge branch 'latex-documentation' of https://bitbucket.org/bigladdersoftware/energyplus/branches into latex-documentation

matthew-larson committed:
Fix table headers.

matthew-larson committed:
Add missing content for using-energyplus-for-compliance.

matthew-larson committed:
Add missing table.

matthew-larson committed:
Minor fixes.

matthew-larson committed:
Fix table headers.

michael-okeefe committed:
Finish diffing in corrections for engineering reference

super/subscripts, tables, equations, missing content, etc

michael-okeefe committed:
Merge branch 'latex-documentation' of https://bitbucket.org/bigladdersoftware/energyplus into latex-documentation

matthew-larson committed:
Add missing content.

michael-okeefe committed:
Merge branch 'latex-documentation' of https://bitbucket.org/bigladdersoftware/energyplus into latex-documentation

matthew-larson committed:
Fix tables headers, subscripts, and add missing content.

michael-okeefe committed:
Fix issue with XeLaTeX compiler hanging if \tabularnewline doesn't appear after caption...

michael-okeefe committed:
Merge branch 'latex-documentation' of https://bitbucket.org/bigladdersoftware/energyplus into latex-documentation

michael-okeefe committed:
Include missing tables

Myoldmopar commented:
Windows CI appears stuck.  It was churning away happily all day, and then boom.  I'm wondering if it hit some API limit with so much activity today.  Windows has never had any issue building this branch, so I'm not worried about that.  I'm merging this in.  I'll then get to work on @EnergyArchmage 's pull request(s) to get the doc changes ported over.

Thanks for a **huge** effort, @nealkruis @michael-okeefe @matthew-larson.  This is a massive improvement in our workflow and ability to create packages and excellent documentation.  I'll keep working through the docs over time and fixing misc things like image sizes and over- and under-full boxes.

FYI @lgentile @amirroth @jasondegraw @mitchute 

![image](https://cloud.githubusercontent.com/assets/849824/13973671/f36b3bf0-f06a-11e5-8831-ff75aeeab661.png)

mitchute commented:
:clap: :exclamation:
